Modix.Bot/Extensions/ClaimsExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using System.Text;
+using Modix.Data.Models.Core;
+
+namespace Modix.Bot.Extensions
+{
+    public static class ClaimsExtensions
+    {
+        /// <summary>
+        /// Turns a list of <see cref="AuthorizationClaim" /> into a newlined list of claims.
+        /// </summary>
+        /// <param name="claims">Claims to turn into strings and add to the message</param>
+        /// <returns>Claims as a string list, newlined.</returns>
+        public static string ToMessageableList(this IReadOnlyCollection<AuthorizationClaim> claims)
```

Comment:
Presumably, this is here because you intend to use it elsewhere in the codebase, not just in `AuthorizationModule`? Where else in the bot are we rendering a list of claims for the user?

Also, I vote naming this `.Render()`.

Reply:
Yeah I was wary about this. I implemented this instead of the `string.Join(hardcodedNewLine, ....)` approach we were using. This is a bit anemic and I prefer the original approach to be honest.

Reply:
I mean, implement it how you want, but the question here is, "Does it need to be a separate extension method?" Seems like the answer right now is "No."

Reply:
Agreed. Removed.